variables.tf

```diff
@@ -10,6 +10,48 @@ variable "instance_name" {
   default     = ""
 }
 
+variable "wd_instance_type" {
+  description = "Waggle Dance instance type, possible values: ecs,ec2."
+  type        = "string"
+  default     = "ecs"
+}
+
+variable "waggledance_version" {
+  description = "Waggle Dance verstion to install on EC2 nodes."
+  type        = "string"
+  default     = "3.3.2"
+}
+
+variable "ami_id" {
+  description = "Amazon Linux AMI, when using ec2 instance type for waggle-dance."
```

Comment:
"when using ec2 instance type for waggle-dance." --> "when using ec2 instance type for Waggle Dance."